1.0/tools/tests.md

```diff
@@ -59,9 +59,9 @@ Follow the steps below to get set up, or watch the Polycast:
 
         npm install -g web-component-tester
 
-    On Mac OS X you need to [manually install][selenium] the latest Safari 
-    extension for Selenium. See the 
-    [Web Component Tester Polycast][workaround-example] for 
+    On Mac OS X you need to [manually install][selenium] the latest SafariDriver
```

Comment:
I think "SafariDriver" is the actual name of the Safari extension, and I was trying to be a little more explicit/helpful. Since I'm already indicating the filename (SafariDriver.safariextz) in the same sentence, I guess this change is overkill. I can restore this to "Safari" if you'd prefer.